test/money.html

```diff
@@ -84,20 +83,42 @@
 		});
 
 		test('renderAmount ', () => {
-			assert.equal(renderAmount({ amount }), undefined);
-			assert.isObject(locAmount, 'testamount is configured for locale');
-			testCurrencies.forEach(currency => {
-				const output = locAmount[alt ? currency + '2' : currency] || locAmount[currency];
-				assert.isString(output, `testcurrency is configured for testamount - ${alt} ${isEdge} ${isSafari}`);
-				assert.equal(renderAmount({ amount, currency }), output);
-			});
 			assert.equal(renderAmount(null), undefined);
 			assert.equal(renderAmount(), undefined);
-			assert.equal(renderAmount({ amount, currency: 'USD' }), renderMoney({ amount, currency: 'USD' }));
+			assert.equal(typeof renderAmount({ amount: 100, currency: 'USD'}), 'string');
+			assert.isAtLeast(renderAmount({ amount: 100, currency: 'USD'}).length, 4);
+		});
+
+		Object.entries(localizedOutputs).forEach(([locale, localizedOutput]) => {
+			const locAmount = localizedOutput[amount];
+
+			test('renderAmount ', () => {
+				assert.equal(renderAmount({ amount }, locale), undefined);
+				assert.isObject(locAmount, 'testamount is configured for locale');
+
+				testCurrencies.forEach(currency => {
+					const output = locAmount[alt ? currency + '2' : currency] || locAmount[currency];
+					assert.isString(output, `testcurrency is configured for testamount - ${alt} ${isEdge} ${isSafari}`);
+					assert.equal(renderAmount({ amount, currency}, locale), output);
+				});
+				assert.equal(
+					renderAmount({
+						amount, currency: 'USD'
+					}, locale),
+					renderMoney({
+						amount, currency: 'USD'
+					}, locale)
+				);
+			});
+
+			test('renderNumberAmount ', () => {
+				assert.equal(renderNumberAmount({ amount, currency: 'USD' }, locale), locAmount.number);
+			});
 		});
 
 		test('renderNumberAmount ', () => {
-			assert.equal(renderNumberAmount({ amount, currency: 'USD' }), locAmount.number);
+			assert.equal(typeof renderNumberAmount({ amount: 100, currency: 'USD' }), 'string');
+			assert.isAtLeast(renderAmount({ amount: 100, currency: 'USD'}).length, 3);
```

Comment:
```suggestion
			assert.isAtLeast(renderNumberAmount({ amount: 100, currency: 'USD'}).length, 3);
```

Reply:
Yes, that was the intention, changed to it now.